CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ Improvements:
 
   - PHPStan: Support setting custom config path and memory limit @ungrim97
 
+Breaking changes:
+
+  - Drop support for PHP 8.0. Minimum version is now 8.0
```

Comment:
Did you mean 8.1 is minimum now?

Reply:
yes ... I fixed that in master  :slightly_smiling_face: